diff -up cmt_1.17/src/Makefile.1~ cmt_1.17/src/Makefile
--- cmt_1.17/src/Makefile.1~	2021-01-07 00:05:11.033632924 +0100
+++ cmt_1.17/src/Makefile	2021-01-07 00:05:43.326869297 +0100
@@ -11,7 +11,7 @@ INSTALL_PLUGINS_DIR	=	/usr/lib/ladspa/
 # GENERAL
 #
 
-CFLAGS		=	$(INCLUDES) -Wall -Werror -O2 -fPIC
+CFLAGS		=	$(INCLUDES) -Wall -Werror $(OPTFLAGS) -fPIC
 CXXFLAGS	=	$(CFLAGS)
 PLUGIN_LIB	=	../plugins/cmt.so
 
